lib/CybsNameValuePairClient.php

```diff
@@ -11,9 +11,11 @@
 class CybsNameValuePairClient extends CybsClient
 {
 
-    function __construct($options=array())
+    function __construct($options=array(), $properties=array())
```

Comment:
That is true, but also you can lock your composer to use only the version that works for you, and later make the update. This allows the project to continue evolving, I think